doc/en/components/charts/chart-features-trendlines.md

```diff
@@ -7,7 +7,87 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Trendlines
 
-<!--
+The $PlatformShort$ financial stock chart has support for trendlines, which help to identify a trend in data bound to a series.
 
-data-chart-series-trendlines.md
--->
\ No newline at end of file
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/data-chart-series-trendlines" 
+           alt="$PlatformShort$ Trendlines Example" 
+           github-src="charts/data-chart/series-trendlines">
+</code-view>
+
+<div class="divider--half"></div>
+
+Trendlines in the $PlatformShort$ data chart component are not on by default. In order to set them, you can set the `TrendLineType` property on the corresponding series that you wish the trendline to show up on. You can modify multiple appearance properties of the trendline such as its brush, period, thickness, and dash array.
+
+Trendlines are supported on all series except for stacked and range series. Below is a list of trendlines that can be used with the series of the data chart:
+
+- `None`
+- `CubicFit`
+- `CumulativeAverage`
+- `ExponentialAverage`
+- `ExponentialFit`
+- `LinearFit`
+- `LogarithmicFit`
+- `ModifiedAverage`
+- `PowerLawFit`
+- `QuadraticFit`
+- `QuarticFit`
+- `QuinticFit`
+- `SimpleAverage`
+- `WeightedAverage`
+
+The following code snippet demonstrates how to add a trendline to a series in the $PlatformShort$ data chart component:
+
+```razor
+<FinancialChart 
+    Width="100%"
+    Height="100%"
+    ChartType=FinancialChartType.Bar
```

Comment:
missing double quotes for value of ChartType prop

Reply:
pending